stream/src/main/scala/org/apache/pekko/stream/javadsl/Source.scala

```diff
@@ -2541,6 +2541,45 @@ final class Source[Out, Mat](delegate: scaladsl.Source[Out, Mat]) extends Graph[
         (resource, out) => f(resource, out),
         resource => close.apply(resource).toScala))
 
+  /**
+   * Transform each stream element with the help of a [[AutoCloseable]] resource and close it when the stream finishes or fails.
+   *
+   * The resource creation function is invoked once when the stream is materialized and the returned resource is passed to
+   * the mapping function for mapping the first element. The mapping function returns a mapped element to emit
+   * downstream. The returned `T` MUST NOT be `null` as it is illegal as stream element - according to the Reactive Streams specification.
+   *
+   * The [[AutoCloseable]] resource is closed only once when the upstream or downstream finishes or fails.
+   *
+   * Early completion can be done with combination of the [[takeWhile]] operator.
+   *
+   * Adheres to the [[ActorAttributes.SupervisionStrategy]] attribute.
+   *
+   * You can configure the default dispatcher for this Source by changing the `akka.stream.materializer.blocking-io-dispatcher` or
```

Comment:
`akka.stream.materializer` should be pekko.stream.materializer`

Reply:
✅ updated

Reply:
don't fix this - this is not your mistake - I will do a PR fixing the legacy bug in the docs

Reply:
I created #1057

Reply:
oh I got it. thanks for your fix!

Reply:
the fix is now Merged, please do a rebase, sorry for the bother.

Reply:
✅ rebased to latest main branch.